### DISCUSSION TITLE
长连接异常断开

### DISCUSSION THREAD

**Discussion by beebol** [bfenetworks/bfe#1125] [Q&A (问题和回答)]
**Describe the bug**
新上线一个长连接服务，使用的是SSE与后端长连接通信，连接会出现大概1分钟左右就会断掉。直连后端的nginx是正常的。 

bfe版本：bfe_1.5.0_linux_amd64

**Screenshots**
<img width="1333" alt="image" src="https://github.com/bfenetworks/bfe/assets/4249023/7cbca816-5132-4f73-a435-0198b3f740d5">

后端nginx error日志会报一个异常：
```
2023/09/06 18:01:11 [error] 78284#78284: *212061 upstream prematurely closed connection while reading upstream, client: x.x.x.13, server: api.open
.wutopia.xyz, request: "GET /spaceapi/sse/createConnect?taskType=audio_tools&taskUUID=WS_O230742511173101B20001_amjphsrygmgzrepvnten4rdjqqirqa2if HTTP
/1.1", upstream: "http://127.0.0.1:8012/spaceapi/sse/createConnect?taskType=audio_tools&taskUUID=WS_O230742511173101B20001_amjphsrygmgzrepvnten4rdjqqi
rqa2if", host: "api.open.wutopia.xyz", referrer: "https://www.wutopia.xyz/"
```

**Comment by youerning** [comment]
bfe这个时间点有报错的日志么？

**Comment by mileszhang2016** [comment]
建议看一下BFE的超时设置
https://github.com/baidu/bfe-book/blob/version1/design/timeout/timeout.md
看起来应该某个超时被触发了
